### DISCUSSION TITLE
Warning: Using NumPy C-API based implementation for BLAS functions

### DISCUSSION THREAD

**Discussion by Ander-MZ** [aesara-devs/aesara#1039] [General]
## Description of your problem or feature request

When importing **aesara**, I get a warning that NumPy API will be used for BLAS functions. I have google around a lot and everything points to a linking issue or some flags, but honestly I understand basically nothing about flags and such.

Could someone point me as to how can I correctly configure my env to use the library? I include a file with my env's details.

**Please provide a minimal, self-contained, and reproducible example.**
```python
import aesara
```

**Please provide the full traceback of any errors.**
```python
Python 3.9.13 | packaged by conda-forge | (main, May 27 2022, 17:00:52) 
[Clang 13.0.1 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import aesara
WARNING (aesara.tensor.blas): Using NumPy C-API based implementation for BLAS functions.
```

**Please provide any additional information below.**

I checked that my env's BLAS library is MKL:

The command ```readlink /opt/anaconda3/envs/pyMC/lib/libblas.3.dylib``` outputs: libmkl_rt.dylib

## Versions and main components

* Aesara version: 2.7.3
* Aesara config: See attached file
* Python version: 3.9.13
* Operating system: MacOS 11.15.2
* How did you install Aesara: conda

[aesara_config.txt](https://github.com/aesara-devs/aesara/files/9040877/aesara_config.txt)

[myenv.txt](https://github.com/aesara-devs/aesara/files/9040920/myenv.txt)

**Comment by brandonwillard** [comment]
First, make sure to check the [installation instructions](https://aesara.readthedocs.io/en/latest/install.html); those may provide some important OS-specific installation details.

Also, you should be able to use the `environment.yml` file in the root directory to construct a working Conda environment.  

Aside from that, there's often not much we can do to help with installations in custom environments.  Those kinds of issues are too often very OS and local setup-dependent, making them difficult to reproduce locally and debug.  You're still welcome to explain your issues in a Discussion like this, though.
